packages/help-center/src/hooks/use-context-based-search-mapping.tsx

```diff
@@ -25,12 +27,18 @@ export function useContextBasedSearchMapping( currentRoute: string | undefined )
 	// Find exact URL matches
 	const exactMatch = urlMapping[ currentRoute as keyof typeof urlMapping ];
 	if ( exactMatch ) {
-		return exactMatch;
+		return { contextSearch: exactMatch };
 	}
 
 	// Fuzzier matches
 	const urlMatchKey = Object.keys( urlMapping ).find( ( key ) => currentRoute?.startsWith( key ) );
 	const urlSearchQuery = urlMatchKey ? urlMapping[ urlMatchKey as keyof typeof urlMapping ] : '';
+	const tailoredArticles = urlMatchKey
+		? tailoredArticlesMapping[ urlMatchKey as keyof typeof tailoredArticlesMapping ]
+		: [];
```

Comment:
From my understanding, we prioritize values from urlMapping. Instead, we need to use it as a fallback and prioritize tailored articles

Reply:
Actually, in this hook this is not relevant. I am prioritizing https://github.com/Automattic/wp-calypso/pull/88808/files#diff-332fff2cea84c41b38afcad559fe76d51fd58dcf07a103c143e1a9edc065e1aeR74 the tailored ones by ordering the response arrays after requesting data